src/DependencyInjection/Configuration.php

```diff
@@ -141,6 +141,11 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->info('Whether or not to wrap migrations in a single transaction.')
                     ->defaultTrue()
                 ->end()
+                ->arrayNode('connections')
+                    ->info('A list of connections managed by migrations, for which the migrations administrative table shall be filtered. The "default" connection is automatically filtered.')
+                    ->prototype('scalar')->end()
+                    ->defaultValue([])
+                ->end()
```

Comment:
Having to re-specify connections here feels clumsy. Users won't understand why they need to do so.